### PR TITLE
Two features Bob asked for

### PR DESCRIPTION
**Pull request by EliAndrewC**
This implements 2 features:

1) If you're in at-the-con mode then clicking on the "Return to Attendees" link takes you to a search that displays only the attendee you were just viewing, which makes it easier to check them in.

2) On the admin form, the age group label is displayed next to the birthdate, e.g. it'll say ``(over 21)``, etc.

### CODE REVIEW

File: uber/templates/registration/form.html [hunk 1]
```diff
@@ -178,15 +178,6 @@
...
 
-<!-- TODO: figure this out, maybe
-{% if attendee.badge_type == c.SUPPORTER_BADGE %}
-    <tr>
-        <td> Affiliate: </td>
-        <td> <input type="text" name="affiliate" value="{{ attendee.affiliate|default:"" }}" /> </td>
-    </tr>
-{% endif %}
--->
-
```
Comment (EliAndrewC) [inline uber/templates/registration/form.html]: This commented out code has been sitting here for awhile, so I deleted it while I was looking at the reg form anyway.

File: uber/templates/registration/menu.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+    {% nav_menu attendee "form?id={id}" "Attendee Data" True "shifts?id={id}" "Shifts" attendee.staffing "history?id={id}" "History" True "watchlist?attendee_id={id}" "Watch List Entry" attendee.banned "index" "Return to Attendee List" c.PRE_CON "index?search_text={id}" "Return to Attendee List" c.AT_THE_CON "index" "Return to Attendee List" c.POST_CON %}
 {% else %}
-{% nav_menu attendee "form?id={id}" "Attendee Data" True "shifts?id={id}" "Shifts" attendee.staffing "history?id={id}" "History" True "staffers" "Return to Staffer List" attendee.staffing "index" "Return to Attendee List" True %}
-{% endif %}
\ No newline at end of file
+    {% nav_menu attendee "form?id={id}" "Attendee Data" True "shifts?id={id}" "Shifts" attendee.staffing "history?id={id}" "History" True "index" "Return to Attendee List" c.PRE_CON "index?search_text={id}" "Return to Attendee List" c.AT_THE_CON "index" "Return to Attendee List" c.POST_CON %}
+{% endif %}
```
Comment (EliAndrewC) [inline uber/templates/registration/menu.html]: Every time I do anything with the ``nav_menu`` tag, I remember that we need to replace it with something better.  Ah well, low priority.

File: uber/templates/regform.html [hunk 3]
```diff
@@ -258,7 +258,12 @@
     {% if c.COLLECT_EXACT_BIRTHDATE %}
         <label for="birthdate" class="col-sm-2 control-label">Date of Birth</label>
         <div class="col-sm-6">
-            <input type='text' class="form-control date" name="birthdate" value="{{ attendee.birthdate|date:"Y-m-d"}} "/>
+            <nobr>
+                <input type='text' class="form-control date" name="birthdate" value="{{ attendee.birthdate|date:"Y-m-d"}} "/>
+                {% if c.PAGE_PATH == '/registration/form' and attendee.birthdate %}
+                    ({{ attendee.age_group_conf.desc }})
```
Comment (kitsuta) [inline uber/templates/regform.html]: Can we actually add this to the check_in page as well? Most people won't be clicking through to Attendee Details.

Comment (EliAndrewC) [inline uber/templates/regform.html]: Yes, sorry I forgot to mention it, but I'm gonna do that as part of a separate PR, since I need to make some performance improvements to the check-in form tonight anyway.

**Comment by kitsuta** [comment]
+1
